### ISSUE TITLE
Simplify this package to be just a collection SVG files

### ISSUE THREAD

**Issue by MananTank** [thirdweb-dev/chain-icons#7]
## What?

* Instead of creating a lot of components for React and exporting them, Just have a folder of SVG files which can be imported by users of the library

## Pros

* This will result in reduced JS bundle size for the users of this library 
* This will Simplify the code for this package - no need to have scripts or src or any code at all
* This will allow this package to be used in any framework (or no framework) 

---

I have already created an alternate version of this package - Checkout [chain-icons](https://github.com/MananTank/web3-chain-icons), and [live demo](https://codesandbox.io/s/react-chain-icons-example-fh66gy?file=/src/App.js) using this package

**Comment by jnsdls** [comment]
yeah that approach may be sufficient. It would have to be a major version bump to change away from providing react components
